docs/geedocs/docsrc/answer/fusionAdministration/relocateAssetRoot.rst

```diff
@@ -48,7 +48,7 @@ Relocate your asset root
          command, the tool auto-detects the problems that need to be
          repaired and fixes them.
 
-         ``sudo /opt/google/bin/geconfigureassetroot --repair new_path``
+         ``sudo /opt/google/bin/geconfigureassetroot --repair --chown new_path``
```

Comment:
There is a call to `geconfigurepublishroot` below on line 69. Do we need to add the `--chown` option there as well?

Reply:
I left it out because step says, "Create your new publish root", and if it's being created then the directories will be created with the correct owner and permissions without needing to chown.

Reply:
I left it out on line 69 because it says "Create your new publish root", and it creates it with the correct owner and permissions without needing to `chown`.